openedx/features/course_experience/__init__.py

```diff
@@ -15,9 +15,6 @@
 # Waffle flag to enable the separate course outline page and full width content.
 COURSE_OUTLINE_PAGE_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'course_outline_page', flag_undefined_default=True)
 
-# Waffle flag to enable a single unified "Course" tab.
-UNIFIED_COURSE_TAB_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'unified_course_tab', flag_undefined_default=True)
```

Comment:
There are still some references to `UNIFIED_COURSE_TAB_FLAG` in the docstrings:
- [first](https://github.com/edx/edx-platform/blob/a1ed944c72d5fce14748463a9c3e2c6c13c400ba/openedx/core/djangoapps/waffle_utils/__init__.py#L14-L29),
- [second](https://github.com/edx/edx-platform/blob/9cf2f9f298e5e8be3b3abcaadaf0b7a96d0de0df/openedx/core/djangoapps/waffle_utils/testutils.py#L22-L32).